packages/labs/analyzer/src/lib/javascript/packages.ts

```diff
@@ -19,13 +19,13 @@ export const getPackageRootForModulePath = (
   const {fs, path} = analyzer;
   let searchDir = modulePath as string;
   const root = path.parse(searchDir).root;
```

Comment:
This is a minor fix for finding a package.json right in the filesystem root, i.e. `/package.json` (hit this with the in-memory tests). The loop bailed as soon as we hit the root, before checking in there.

Note, this and the next function are similar to these used internally in TS, but they are marked as `/** @internal */`: https://github.com/microsoft/TypeScript/blob/221cf55a21e448bd3fe2cf26a754c9c0dda3dca3/src/compiler/moduleNameResolver.ts#L1825-L1882